src/System.Management.Automation/engine/MshMemberInfo.cs

```diff
@@ -4224,9 +4239,14 @@ internal int Count
         {
             get
             {
-                lock (_members)
+                if (_members == null)
+                {
+                    return 0;
+                }
+
+                lock (Members)
                 {
-                    return _members.Count;
+                    return Members.Count;
```

Comment:
Same nitpick here, how about just use `_members` for the `lock(...)` block?

Reply:
Fixed.